web/html/src/manager/storybook/storybook.tsx

```diff
@@ -14,7 +14,7 @@ const STORAGE_KEY = "storybook-show-code";
 export const Storybook = () => {
   const [_hash, setHash] = useState(window.location.hash);
   const hash = _hash.replace(/^#/, "");
-  const normalize = (input: string = "") => input.replaceAll(" ", "-").toLowerCase();
+  const normalize = (input: string = "") => input.replace(" ", "-").toLowerCase();
```

Comment:
Replace does not work the same way in Javascript as it does in Java, this would only replace the first occurrence, not all of them.